moveit_simple/test/moveit_simple.cpp

```diff
@@ -453,6 +453,57 @@ TEST_F(UserRobotTest, kinematics)
 }
 
 
+TEST_F(UserRobotTest, custom_tool_link)
```

Comment:
This test looks like a copy and paste of the test above.  Is there a better way to reuse code?  Gtest supports scoped tests (i.e. extracting tests into functions) or test fixtures.

Reply:
I can look into that. I've also added tests for execute(...) but I haven't commited those changes yet.